### DISCUSSION TITLE
Can dpgen disturb the simulation box in the specified direction?

### DISCUSSION THREAD

**Discussion by ChengQIAN93** [deepmodeling/dpgen#555] [Q&A]
Hi developer,

  I recently wanted to use the dpgen to disturb the simulation box in specific directions, such as x-direction or XY-directions. But I don’t know whether dpgen can specify the direction of the disturbance. So if it can do this work, please tell me how to set it up in the param file. Thank you !

Best,
Cheng

**Comment by AnguseZhang** [comment]
As far as I know, currently DP-GEN codes do not support your needs. This needs some implementation.

**Reply by ChengQIAN93** [reply]
thank you for your reply. Can this function be added to dpgen in the future? I think this is useful for 2D materials.
